acceptance/tests/aix/aix_package_provider.rb

```diff
@@ -22,6 +18,11 @@ def assert_package_version(package, expected_version)
 version1 = '1.7.10.4'
 version2 = '1.8.6.4'
 
+teardown do
+  on hosts, "rm -rf #{dir}"
+  on hosts, puppet('resource', 'package', "'#{package}' ensure=absent")
```

Comment:
I'm surprised that we're quoting the package, e.g. `puppet resource package 'mypackage' ensure=absent`? Is that actually removing the package? Or is it trying to remove a package whose name contains quotes, and since there's no package with that name, doing nothing?